### PR TITLE
Development

### PR DESCRIPTION
**Pull request by KevinMayfield**
Mostly admin but 

NHSDigital-Claim has been changed to move to the same profile to PractitionerRole as Task and MedicationDispense

https://fhir.nhs.uk/StructureDefinition/NHSDigital-PractitionerRole-SDS

### CODE REVIEW

File: ValueSet/NHSDigital-DocumentType.xml [hunk 1]
```diff
@@ -1,7873 +0,0 @@
-<ValueSet xmlns="http://hl7.org/fhir">
```
Comment (omarisgreat) [inline ValueSet/NHSDigital-DocumentType.xml:1]: What is the rationale behind deleting the ValueSet, is it no longer used/replaced by UKCore etc.?

File: Task/Task-medication-repeat-reorder.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
         ]
     },
     "focus": {
-        "reference": "MedicationRequest/ee86a018-779c-4809-999f-a9d89cdfd30f"
```
Comment (omarisgreat) [inline Task/Task-medication-repeat-reorder.json:26]: Do relative links for resource references no longer work or is this a QoL change?

File: Task/Task-ers-Triage-reviewDeferral.json [hunk 3]
```diff
@@ -31,7 +31,7 @@
     "resourceType": "Task",
     "focus": {
         "type": "ServiceRequest",
-        "reference": "ServiceRequest/be735573-9f24-42e1-ae98-af89d632e4bf"
```
Comment (omarisgreat) [inline Task/Task-ers-Triage-reviewDeferral.json:34]: Same comment as the medication repeat order task, is this change required?

File: StructureDefinition/NHSDigital-Claim.StructureDefinition.xml [hunk 4]
```diff
@@ -611,11 +614,6 @@
       <path value="Claim.item.detail.programCode.coding.system" />
       <fixedUri value="https://fhir.nhs.uk/CodeSystem/medicationdispense-endorsement" />
     </element>
-    <element id="Claim.item.detail.programCode:dispensingEndorsement.text">
```
Comment (omarisgreat) [inline StructureDefinition/NHSDigital-Claim.StructureDefinition.xml:614]: Why was this removed? It is useful guidance for dispensers submitting claims on where they need to provide further information. Are you expecting this to be moved elsewhere?

File: StructureDefinition/NHSDigital-Claim.StructureDefinition.xml [hunk 5]
```diff
@@ -566,7 +563,7 @@
...
+      <definition value="Dispensing Endorsement  - Details of the endorsement(s) to support the claim for this medication item. &#xD;&#xA;&#xD;&#xA;Charge Payment - Details on whether a prescription charge was paid for the medication treatment.&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&#xD;&#xA;&quot;programCode&quot;:  [&#xD;&#xA;    {&#xD;&#xA;        &quot;coding&quot;:  [&#xD;&#xA;            {&#xD;&#xA;                &quot;system&quot;: &quot;https://fhir.nhs.uk/CodeSystem/DM-prescription-charge&quot;,&#xD;&#xA;                &quot;code&quot;: &quot;paid-once&quot;,&#xD;&#xA;                &quot;display&quot;: &quot;Paid Once&quot;&#xD;&#xA;            }&#xD;&#xA;        ]&#xD;&#xA;    },&#xD;&#xA;    {&#xD;&#xA;        &quot;coding&quot;:  [&#xD;&#xA;            {&#xD;&#xA;                &quot;system&quot;: &quot;https://fhir.nhs.uk/CodeSystem/medicationdispense-endorsement&quot;,&#xD;&#xA;                &quot;code&quot;: &quot;IP&quot;,&#xD;&#xA;                &quot;display&quot;: &quot;Invoice Price&quot;&#xD;&#xA;            }&#xD;&#xA;        ]&#xD;&#xA;    }&#xD;&#xA;],&#xD;&#xA;&#xD;&#xA;```" />
```
Comment (omarisgreat) [inline StructureDefinition/NHSDigital-Claim.StructureDefinition.xml:566]: Where do would you like to see guidance on additional required information required on dispensing endorsements? This was to make the requirement clear to dispensers using the IG

File: StructureDefinition/NHSDigital-Claim.StructureDefinition.xml [hunk 6]
```diff
@@ -156,7 +141,7 @@
...
...;id&quot;: &quot;provider&quot;,&#xD;&#xA;            &quot;practitioner&quot;: {&#xD;&#xA;                &quot;identifier&quot;: {&#xD;&#xA;                    &quot;system&quot;: &quot;https://fhir.hl7.org.uk/Id/gphc-number&quot;,&#xD;&#xA;                    &quot;value&quot;: &quot;2344242&quot;&#xD;&#xA;                },&#xD;&#xA;                &quot;display&quot;: &quot;|Bob||Buller|&quot;&#xD;&#xA;            },&#xD;&#xA;            &quot;organization&quot;: {&#xD;&#xA;                &quot;identifier&quot;: {&#xD;&#xA;                    &quot;system&quot;: &quot;https://fhir.nhs.uk/Id/ods-organization-code&quot;,&#xD;&#xA;                    &quot;value&quot;: &quot;FA976&quot;&#xD;&#xA;                },&#xD;&#xA;                &quot;display&quot;: &quot;Bob's Pharmacy&quot;&#xD;&#xA;            },&#xD;&#xA;            &quot;telecom&quot;: [&#xD;&#xA;                {&#xD;&#xA;                    &quot;system&quot;: &quot;phone&quot;,&#xD;&#xA;                    &quot;use&quot;: &quot;work&quot;,&#xD;&#xA;                    &quot;value&quot;: &quot;01234567890&quot;&#xD;&#xA;                }&#xD;&#xA;            ]&#xD;&#xA;        }&#xD;&#xA;    ],&#xD;&#xA;```" />
```
Comment (omarisgreat) [inline StructureDefinition/NHSDigital-Claim.StructureDefinition.xml:144]: I can't tell in the GitHub diff whether this renders correctly but I'll assume it does

**Review by omarisgreat — COMMENTED** [review]
Added a few questions against some changes (especially guidance in the claim profile)

File: ValueSet/NHSDigital-DocumentType.xml (hunk 1 above)
Comment (KevinMayfield) [reply ValueSet/NHSDigital-DocumentType.xml:1]: Yes it's a duplicate of the UKCore valueSet

File: Task/Task-medication-repeat-reorder.json (hunk 2 above)
Comment (KevinMayfield) [reply Task/Task-medication-repeat-reorder.json:26]: This is for workflow elaboration and demonstrating using a shared repository

File: StructureDefinition/NHSDigital-Claim.StructureDefinition.xml (hunk 4 above)
Comment (KevinMayfield) [reply StructureDefinition/NHSDigital-Claim.StructureDefinition.xml:614]: Will investigate, could be a forge change

File: Task/Task-ers-Triage-reviewDeferral.json (hunk 3 above)
Comment (KevinMayfield) [reply Task/Task-ers-Triage-reviewDeferral.json:34]: This is for workflow elaboration and demonstrating using a shared repository

**Comment by omarisgreat** [comment]
Approved, you can merge at your leisure
